components/CatModal/useCatActions.tsx

```diff
@@ -130,10 +130,16 @@ const useCatActions = (catId: string) => {
       await addImages(zoneId, catId, formBody);
     }
     mutate(`${URL}${zoneId}/${catId ?? catID}/images`);
-    mutate(`${URL}${zoneId}`);
-    mutate(URL);
   }
 
-  return { AddCat, UpdateCat, DeleteCat, SterilizeCat, GetImages, AddImages };
+  const CompressImage = async (file: File): Promise<File> =>  {
+    const options = {
+      maxSizeMB: 1
```

Comment:
Please move hardcoded value to a constant. The name of the constant can help document it purpose as well.